SOURCES/etc/xapi.d/plugins/netdata.py

```diff
@@ -53,21 +53,16 @@ def install_netdata(session, args):
     destination = args['destination']
     with OperationLocker():
         temp_dir = tempfile.mkdtemp()
-        files = []
         try:
-            command = ['yum', 'install', '-y', 'netdata', '--enablerepo=xcp-ng-testing']
-            result = run_command(command)
-            if result['exit'] == 0:
-                with open("/etc/netdata/stream.conf", "w") as conf_file:
-                    conf_file.write(
-                        netdata_streaming_content.format(destination, api_key))
-                result2 = run_command(['service', 'netdata', 'restart'])
-                if result2['exit'] == 0:
-                    return json.dumps(True)
-                else:
-                    raise Exception(result2)
+            install_package('netdata')
+            with open("/etc/netdata/stream.conf", "w") as conf_file:
+                conf_file.write(
+                    netdata_streaming_content.format(destination, api_key))
+            result2 = run_command(['service', 'netdata', 'restart'])
```

Comment:
Since there's no `result` variable anymore, people from the future might wonder why this one is named `result2`